### PR TITLE
IIS Metadata

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
This PR adds IIS metrics to `windows.plugin` metadata. It also fixes wrong pair of dimensions in `go.d.plugin`.

##### Test Plan

1. Take a look in CI
2. Verify if something is missed.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

File: src/collectors/windows.plugin/metadata.yaml [hunk 1]
```diff
@@ -274,3 +274,150 @@ modules:
               chart_type: line
               dimensions:
                 - name: temperature
+  - meta:
+      plugin_name: windows.plugin
+      module_name: PerflibWebService
+      monitored_instance:
+        name: IIS (Internet Information Services) statistics
```
Comment (ilyam8) [inline src/collectors/windows.plugin/metadata.yaml:281]:
```suggestion
        name: IIS
```

File: src/collectors/windows.plugin/metadata.yaml [hunk 2]
```diff
@@ -274,3 +274,150 @@ modules:
               chart_type: line
               dimensions:
                 - name: temperature
+  - meta:
```
Comment (Ancairon) [inline src/collectors/windows.plugin/metadata.yaml:277]: @thiagoftsm is this PR covering only IIS? If it is supposed to fix the whole metadata YAML, then I see more stuff in the directory that I don't know if they are covered.

PerflibProcesses
PerflibMemory
PerflibThermalzone

Have been implemented.

PerflibMSSQL
PerflibNetwork
PerflibProcessor
PerflibStorage

I see haven't, will you add them in this PR or in a different one?

File: src/collectors/windows.plugin/metadata.yaml [hunk 3]
```diff
@@ -274,3 +274,150 @@ modules:
...
+        description: ""
+      keywords:
+        - IIS
+        - HTTP
+        - Web service
+      most_popular: false
+    overview:
+      data_collection:
+        metrics_description: |
+          This collector monitors the current number of processes, threads, and context switches on Windows systems.
```
Comment (Ancairon) [inline src/collectors/windows.plugin/metadata.yaml:299]: I believe this is copy-pasted and missed, please change to proper sentence

File: src/collectors/windows.plugin/metadata.yaml [hunk 4]
```diff
@@ -274,3 +274,150 @@ modules:
...
+        method_description: |
+          It queries the 'System' object from Perflib in order to gather the metrics.
```
Comment (Ancairon) [inline src/collectors/windows.plugin/metadata.yaml:301]: Check this too

File: src/collectors/windows.plugin/metadata.yaml [hunk 5]
```diff
@@ -274,3 +274,150 @@ modules:
...
+        list: []
+    alerts: []
+    metrics:
+      folding:
+        title: Metrics
+        enabled: false
+      description: ""
+      availability: []
+      scopes:
+        - name: IIS
```
Comment (Ancairon) [inline src/collectors/windows.plugin/metadata.yaml:350]: @thiagoftsm will this detect if I got two instances of IIS on my system?

File: src/collectors/windows.plugin/metadata.yaml (hunk 2 above)
Comment (thiagoftsm) [reply src/collectors/windows.plugin/metadata.yaml:277]: Hello @Ancairon ,

Yes, I am only working on IIS. 

Today, I’ll start working on the remaining Perflibs we’re missing. However, I plan to address each of them in separate PRs to make your review process easier.

File: src/collectors/windows.plugin/metadata.yaml (hunk 3 above)
Comment (thiagoftsm) [reply src/collectors/windows.plugin/metadata.yaml]: You are right, I am sorry for this mistake. I pushed a commit fixing it.

File: src/collectors/windows.plugin/metadata.yaml (hunk 4 above)
Comment (thiagoftsm) [reply src/collectors/windows.plugin/metadata.yaml]: I pushed another commit changing 'System' to 'Web Service'

**Review by ilyam8 — APPROVED** [review]

**Pull request merged**